src/server/editorServices.ts

```diff
@@ -410,8 +408,8 @@ namespace ts.server {
 
             Debug.assert(!!this.host.createHash, "'ServerHost.createHash' is required for ProjectService");
 
+            this.currentDirectory = this.host.getCurrentDirectory();

```

Comment:
I'm really confused why we use this anywhere. What impact does the folder tsserver is running from have on anything?